components/reports/temperature/ReportTempChart.vue

```diff
@@ -186,9 +244,25 @@ export default {
 					y1: freezing,
 					yref: 'y',
 					line: {
-						width: 1,
-						color: 'rgb(175, 175, 175)',
+						width: 2,
+						color: '#cccccc',
 					},
+					layer: 'below',
+				})
+				layout.annotations.push({
+					x: 1,
+					y: freezing,
+					xref: 'paper',
+					yref: 'y',
+					text: 'Freezing',
```

Comment:
I'm baffled: this doesn't show up on my computer!  See general convo for some more notes.

Reply:
And, turns out, I was just not looking at a chart with the freezing line visible D: hah!  This is fine.